client/include/ClientGame.h

```diff
@@ -81,6 +81,7 @@ class ClientGame
     bool requestRightward = false;
     bool requestJump = false;
     bool requestThrowEgg = false;
+    bool requestSeasonAbility = false;
```

Comment:
Is this ever used?

Reply:
Nope, good catch. I missed that while merging.